app/routes/application-list.js

```diff
@@ -308,6 +314,18 @@ module.exports = router => {
           })
         })
       }
+
+      if (studyModes && studyModes.length) {
+        selectedFilters.categories.push({
+          heading: { text: 'Study modes' },
```

Comment:
Not sure about this text. Trying to avoid jargon. We go for ‘Full time or part time’ elsewhere.

But this is gonna be tricksy. Let’s discuss with @johndoates

Reply:
Changed to "Full time or part time" as suggested.